src/connection-manager/dialer/dial-request.ts

```diff
@@ -102,10 +102,23 @@ export class DialRequest {
           }
         }
 
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
Minor comment, this eslint comment is duplicated here and on the next line.

Reply:
changed

Reply:
If you follow the [suggestions](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md) for the @typescript-eslint/strict-boolean-expressions rule you wouldn't have to disable eslint.

In brief, instead of `if (!conn)` do `if (conn == null)`.

Reply:
changed, thx for the suggestion